Local development build fails

### Observed
I want to do a local build of the HTML version while making changes to the req42 framework.
When I run `gradle asciidoctor` as described in the README, then no output is generated and the command generates this output:

```
gradle asciidoctor
> Task :asciidoctor FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':asciidoctor'.
> Cannot locate a Guava JAR in the Gradle distribution
```

### Expected
The HTML version is generated from the local resources and files are written to the `/build` directory.

### Environment 

**Operating System**
```
**_sw_vers_** 
ProductName:            macOS
ProductVersion:         15.5
BuildVersion:           24F74
```

**Java Platform**
```
gradle --version

------------------------------------------------------------
Gradle 7.6.5
------------------------------------------------------------

Build time:   2025-06-04 13:02:22 UTC
Revision:     13a09d0cc61025596acd4acf05d1980a79838906

Kotlin:       1.7.10
Groovy:       3.0.13
Ant:          Apache Ant(TM) version 1.10.13 compiled on January 4 2023
JVM:          17.0.15 (Eclipse Adoptium 17.0.15+6)
OS:           Mac OS X 15.5 aarch64
```

--- a/build.gradle
+++ b/build.gradle
@@ -3,7 +3,7 @@
 //
 
 plugins {
-    id 'org.asciidoctor.jvm.convert' version '3.3.0'
+    id 'org.asciidoctor.jvm.convert' version '4.0.4'
 }
 
 apply plugin: 'java'

